src/ingestor/writer/vote.ts

```diff
@@ -57,14 +57,25 @@ export async function verify(body): Promise<any> {
   }
 }
 
-export async function action(body, ipfs, receipt, id): Promise<void> {
+export async function action(body, ipfs, receipt, id, context): Promise<void> {
   const msg = jsonParse(body.msg);
   const voter = getAddress(body.address);
   const created = parseInt(msg.timestamp);
   const choice = JSON.stringify(msg.payload.choice);
   const metadata = JSON.stringify(msg.payload.metadata || {});
   const app = msg.payload.app || '';
   const reason = msg.payload.reason || '';
+
+  // Check if voting power is final
+  let withDelegation = false;
+  context.proposal.strategies
+    .map(strategy => strategy.name)
+    .forEach(name => {
+      if (name.includes('delegation')) withDelegation = true;
```

Comment:
Everything looks great 🚀 except that delegation can be used in child strategy too, recently POH used math strategy on top of delegation, maybe there can be grand child strategy too :sweat_smile: (we should limit these levels of strategy) 

I think best solution for now would be to stringify strategies object and check if there is delegation string

For delegation strategy, would be great if there is a way to just update delegate's VP (when delegator vote)